src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -4397,6 +4397,11 @@ public static boolean getAccordTransactionsEnabled()
         return conf.accord_transactions_enabled;
     }
 
+    public static void setAccordTransactionsEnabled(boolean b)
+    {
+        conf.accord_transactions_enabled = b;
```

Comment:
nit: Does `accord_transactions_enabled` need to be volatile now? (I've seen comments on the mailing list around this not actually being required, but I'm not sure I understand. Is it that there are so many other locks held and released around touching these values that they're bound to be visible within a short time anyway?)

Reply:
> nit: Does accord_transactions_enabled need to be volatile now?

since this is tests only, and that runs in `main` thread, I don't believe we need to.  This gets called before C* starts, so before any thread spins up

Reply:
Ah didn't realize it was test-only